conditional/templates/intro_evals.html

```diff
@@ -20,16 +20,21 @@
                     </div>
                     <div class="col-sm-5 col-md-4 col-lg-5">
                         <h4 class="eval-name">{{m['name']}}</h4>
-                        <h6 class="eval-uid">{{m['uid']}}</h6>
+                            {% if not m['ldap_account'] %}
+                                <span class="label label-default">Internal Account: {{m['uid']}}</span>
+                            {% else %}
+                                <h6 class="eval-uid">{{ m['uid'] }}</h6>
+                            {% endif %}
+
                     </div>
                     <!---->
                     <div class="col-sm-5 col-md-6 col-lg-5 vcenter">
                         <div class="row">
                             <div class="text-center">
-                                {% if m['signatures_missed'] > 0 %}
+                                {% if m['signatures_missed'] == 65535 %}
```

Comment:
I don't really like using "magic" numbers. Can we use an enum or just a nil value here to represent this instead?